extensions/ql-vscode/src/remote-queries/view/SortRepoFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { FilterIcon } from '@primer/octicons-react';
+import { ActionList, ActionMenu } from '@primer/react';
+
+export type Sort = 'name' | 'stars' | 'results';
+type SortBy = { name: string, sort: Sort }[];
+type Props = {
+  sort: Sort;
+  setSort: (sort: Sort) => void;
+};
+
+type Sortable = {
+  nwo: string;
+  starCount?: number;
+  resultCount?: number;
+};
+
+const sortBy: SortBy = [
```

Comment:
We have a type and a value both called "sort by", which I found a bit difficult to follow. Perhaps `sorts` is a better name for this one? Or `sortByOptions`?

Reply:
It's the capitalization that should help distinguish a type from a value. But, anyway, the type is gone now. I think it's more understandable now.